src/xtgeo/surface/_regsurf_import.py

```diff
@@ -12,6 +12,7 @@
 from xtgeo import _cxtgeo
 from xtgeo.common import XTGeoDialog, null_logger
 from xtgeo.common.constants import UNDEF_MAP_IRAPA, UNDEF_MAP_IRAPB
+from xtgeo.surface._regsurf_ijxyz_parser import parse_ijxyz
 from xtgeo.surface._zmap_parser import parse_zmap
```

Comment:
You'll probably need to put `from __future__ import annotations` in here as well